src/linagora.esn.videoconference.calendar/app/components/calendar-videoconference-form/calendar-videoconference.less

```diff
@@ -13,3 +13,40 @@
     flex-grow: 1;
   }
 }
+
+.conference-btns {
+  margin-left: 10px;
```

Comment:
Generally, we should use `4px`, `8px`, `12px`, `16px`, etc. for spacing units. It's a bad practice in UI Design to use spacing units freely.